README.md

```diff
@@ -96,28 +96,28 @@ apps. An example:
 }
 ```
 
-**redirects:** you can specify certain url paths to be redirected to another url by supplying configuration to the `redirects` key. Path matching is similar to using custom routes. Redirects use the `301` HTTP status code by default, but this can be overridden by configuration.
+**function():** you can specify certain url paths to be redirected to another url by supplying configuration to the `function()` key. Path matching is similar to using custom routes. function() use the `301` HTTP status code by default, but this can be overridden by configuration.
```

Comment:
Is the change from `"redirects"` to `"function()"` intentional or just a find/replace error?